projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item-list/cart-item-list.component.ts

```diff
@@ -49,13 +59,26 @@ export class CartItemListComponent implements OnInit {
     return this._items;
   }
 
-  constructor(protected cartService: CartService, protected fb: FormBuilder) {}
+  constructor(
+    protected cartService: CartService,
+    protected fb: FormBuilder,
+    protected selectiveCartService: SelectiveCartService,
```

Comment:
As mentioned in other comments. Another potential breaking change

Reply:
updated.